README.md

```diff
@@ -1 +1,95 @@
-# service-spark-iasworld
\ No newline at end of file
+# Spark Extractor for iasWorld
+
+This repository contains the dependencies and code necessary to run
+[Spark](https://spark.apache.org/docs/latest/) extract jobs targeting the
+CCAO's iasWorld system-of-record. It is a replacement for
+[`service-sqoop-iasworld`](https://github.com/ccao-data/service-sqoop-iasworld),
+which is now deprecated.
+
+The Spark jobs pull iasWorld tables (or parts of tables) via
+[JDBC](https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html) and
+write them as [Hive-partitioned](https://duckdb.org/docs/data/partitioning/hive_partitioning.html)
+Parquet files to [AWS S3](https://aws.amazon.com/s3/). The Data Department then
+queries these Parquet files using [AWS Athena](https://aws.amazon.com/athena),
+giving us a 1-1 mirror of the system-of-record for analytical queries.
+
+## Submitting jobs
+
+> [!NOTE]
+> Before attempting to submit jobs to the cluster, first make sure the Spark
+> Docker Compose stack is active by running `docker compose up -d` in the
+> repository.
+
+`service-spark-iasworld` jobs are submitted via JSON, either as a string or
+as a file. All jobs should have the following format:
+
+```json
+{
+  "addn": {
```

Comment:
The "job name" doesn't actually matter. It's not used in any way, just there for reference.